documentation/notes/joss_paper/paper.md

```diff
@@ -68,12 +67,12 @@ However, these models are less often considered in literature, despite their dem
 For an efficient handling of the model by users, its documentation is conceived such that its equations and parameters are explained and linked to the code.
 In the future, its development will be done in a modular fashion which enable the connection of the atmosphere to various other subsystems and its usage with built-in and external toolboxes.
 
-The choice to use Python was specifically made to facilitate its use in Jupyter [@jupyter] notebooks and the multiple recent machine learning libraries that are available in this 
+The choice to use Python was specifically made to facilitate its use in Jupyter [@jupyter] Notebooks and the multiple recent machine learning libraries that are available in this
 language.
 
 # State of the field
 
-Other software might interest the reader in need for an easy-to-use idealized atmospheric model.
+Other software might interest the reader in need of an easy-to-use idealized atmospheric model.
```

Comment:
This is purely a suggestion I am regarding clarity (feel free to ignore it since it won't influence my review outcome) but to me the word 'of' makes this sentence flow better than 'for'. With 'for', as-is, I had to read it through a number of times to try to understand the intention (it could just be me, though). It's quite hard to explain why I think 'of' is clearer and more appropriate, but as an indicator "in need of" is more common that "in need for" (for example it returns ~2x as many hits when searched in Google at the moment).

Reply:
Ok thank you.